rocketpool-cli/node/deposit.go

```diff
@@ -92,23 +92,26 @@ func nodeDeposit(c *cli.Context) error {
 	}
 
 	// Post a warning about fee distribution
-	if !(c.Bool("yes") || cliutils.Confirm(fmt.Sprintf("%sNOTE: by creating a new minipool, your node will automatically claim and distribute any balance you have in your fee distributor contract. If you don't want to claim your balance at this time, you should not create a new minipool.%s\nWould you like to continue?", colorYellow, colorReset))) {
+	if !(c.Bool("yes") || cliutils.Confirm(fmt.Sprintf("%sNOTE: By creating a new minipool, your node will automatically claim and distribute any balance you have in your fee distributor contract. If you don't want to claim your balance at this time, you should not create a new minipool.%s\nWould you like to continue?", colorYellow, colorReset))) {
 		fmt.Println("Cancelled.")
 		return nil
 	}
 
 	// Get deposit amount
 	var amount float64
 
-	// Get deposit amount options
-	amountOptions := []string{
-		"8 ETH",
-	}
-
-	// Prompt for amount
-	selected, _ := cliutils.Select("Please choose an amount of ETH to deposit:", amountOptions)
-	switch selected {
-	case 0:
+	if c.String("amount") != "" {
+		// Parse amount
+		depositAmount, err := strconv.ParseFloat(c.String("amount"), 64)
```

Comment:
i feel like `amount` for a deposit is not a float. it's either 8 or 16, in practice